### PR TITLE
Fixes ENYO-2144

### PR DESCRIPTION
**Pull request by ryanjduffy**
## Issue

Expandable controls are duplicating component tree and features
## Fix

Refactor things for better reuse

Enyo-1.1-DCO-Signed-off-by: Ryan Duffy ryan.duffy@lge.com

### CODE REVIEW

File: lib/ExpandableInput/ExpandableInput.js [hunk 1]
```diff
@@ -180,11 +163,12 @@ module.exports = kind(
...
 	* @private
 	*/
-	currentValueText: function () {
-		return (this.value === '') ? this.noneText : this.value;
+	currentValueShowing: function () {
+		return !this.open && this.currentValueText() !== '' && this.type != 'password';
 	},
 
 	/**
+<<<<<<< HEAD
```
Comment (aarontam) [inline lib/ExpandableInput/ExpandableInput.js]: Some merge conflict remnants. :smile:

File: lib/Accordion/Accordion.js [hunk 2]
```diff
@@ -89,33 +81,5 @@ module.exports = kind(
...
-		]}
-	],
-
-	/**
-	* @private
-	*/
-	bindings: [
-		{from: '.disabled', to: '.$.headerWrapper.disabled'}
-	]
+	classes: 'moon-accordion'
```
Comment (aarontam) [inline lib/Accordion/Accordion.js]: I wonder if, somewhere down the road, we can deprecate this kind and just make this a special case of `ExpandableListItem` i.e. enabling via a property of that kind.

File: lib/ExpandableInput/ExpandableInput.js [hunk 3]
```diff
@@ -294,29 +266,11 @@ module.exports = kind(
...
 			this.focusInput();
 		} else {
-			this.closeDrawerAndHighlightHeader();
+			this.expandContract();
 		}
 		return true;
-	},
-
-	/**
-	* We manually set pointer mode to `false` as this seems to be the
```
Comment (aarontam) [inline lib/ExpandableInput/ExpandableInput.js]: Glad we could get rid of this :+1:

File: lib/LabeledTextItem/LabeledTextItem.js [hunk 4]
```diff
@@ -86,21 +72,18 @@ module.exports = kind(
...
-	},
-	
-	/**
-	* @private
-	*/
-	textChanged: function() {
-		this.$.text.setContent(this.text);
-	}
+	components:[
+		// headerContainer required to avoid bad scrollWidth returned in RTL for certain text widths (webkit bug)
```
Comment (aarontam) [inline lib/LabeledTextItem/LabeledTextItem.js]: Wonder if we should re-verify this behavior with Blink while we're at it.

File: lib/Accordion/Accordion.js (hunk 2 above)
Comment (ryanjduffy) [inline lib/Accordion/Accordion.js]: Agreed. Kept it but marked it deprecated.

File: lib/ExpandableInput/ExpandableInput.js (hunk 3 above)
Comment (ryanjduffy) [inline lib/ExpandableInput/ExpandableInput.js]: Ya, looks like this can be removed. Verified the old bug doesn't repro.

**Pull request merged**